drivers/wireless/ieee80211/bcm43xxx/bcmf_ioctl.h

```diff
@@ -2712,6 +2712,13 @@ typedef struct wl_rssi_event
   int8_t rssi_levels[MAX_RSSI_LEVELS];
 } wl_rssi_event_t;
 
+typedef struct wl_sta_rssi
+{
+  uint32_t          rssi;
+  struct ether_addr sta_addr;
+  uint16_t          foo;
```

Comment:
why name foo?

Reply:
emm... I do not know ... just follow broadcom style
https://lxr.openwrt.org/source/iwinfo/api/broadcom.h#L75